samples/notebooks/McpClient.ipynb

```diff
@@ -0,0 +1,341 @@
+{
+ "cells": [
+  {
+   "metadata": {},
+   "cell_type": "markdown",
+   "source": "# MCP Client Example"
+  },
+  {
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2025-11-13T09:55:27.803922Z",
+     "start_time": "2025-11-13T09:55:27.687527Z"
+    }
+   },
+   "cell_type": "code",
+   "source": [
+    "%use coroutines (1.10.2)\n",
+    "\n",
+    "USE {\n",
+    "    dependencies(\n",
+    "        \"io.modelcontextprotocol:kotlin-sdk-client-jvm:0.7.7\",\n",
+    "        \"io.kotest:kotest-assertions-core-jvm:6.0.4\",\n",
+    "        \"io.ktor:ktor-client-core-jvm:3.2.3\",\n",
+    "        \"io.ktor:ktor-client-logging-jvm:3.2.3\",\n",
+    "        \"io.ktor:ktor-client-apache5-jvm:3.2.3\"\n",
```

Comment:
You can use the http integration for Kotlin notebooks. It’s still the ktor client, but with additional utility functions tailored for notebooks
```
%use ktor-client
```

look here: https://github.com/Kotlin/kotlin-notebook-integrations/blob/master/integrations/http-util/README.md